packages/hdwallet-native/src/arkeo.ts

```diff
@@ -0,0 +1,99 @@
+import { StdTx } from "@cosmjs/amino";
+import { SignerData } from "@cosmjs/stargate";
+import * as core from "@shapeshiftoss/hdwallet-core";
+import * as bech32 from "bech32";
+import CryptoJS from "crypto-js";
+import PLazy from "p-lazy";
+
+import * as Isolation from "./crypto/isolation";
+import { NativeHDWalletBase } from "./native";
+import * as util from "./util";
+
+const ARKEO_CHAIN = "arkeo-mainnet";
```

Comment:
do we know for sure what the mainnet chain id is going to be? this makes sense enough, but just checking